### PR TITLE
Update UG for cs2101

### PR DESCRIPTION
**Pull request by darrtzj**

### CODE REVIEW

**Review by HowSuen — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -308,18 +308,18 @@ to the current date by sorting the list according to the students' Mastery Check
...
-    * `n` (name)
-    * `t` ([Telegram handle](#glossary))
-    * `c` (consultation)
-    * `m` ([Mastery Check](#glossary))
-    * `ra1` ([Reading Assessment 1](#glossary))
-    * `ra2` ([Reading Assessment 2](#glossary))
-    * `pa` ([Practical Assessment](#glossary))
-    * `mt` ([Midterm Test](#glossary))
-    * `ft` ([Final Examination](#glossary))
+    * Name - `n`
```
Comment (HowSuen) [inline docs/UserGuide.md:311]: Can add back the hyperlinks to the glossary

**Review by HowSuen — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (darrtzj) [reply docs/UserGuide.md:311]: Ok! added them back

**Pull request merged**